installing/installing_vsphere/ipi/ipi-vsphere-preparing-to-install.adoc

```diff
@@ -0,0 +1,32 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ipi-vsphere-preparing-to-install"]
+= Preparing to install a cluster using installer-provisioned infrastructure
+include::_attributes/common-attributes.adoc[]
+:context: ipi-vsphere-preparing-to-install
+
+toc::[]
+
+You prepare to install an {product-title} on vSphere by:
```

Comment:
Question: should you use an article if you're just saying "OCP" and not "OCP cluster"? If not, I'd either remove the article like this:
```suggestion
You prepare to install {product-title} on vSphere by:
```
Or add "cluster" like this:
```suggestion
You prepare to install an {product-title} cluster on vSphere by:
```

Reply:
Good catch. Thank you.